src/update.cpp

```diff
@@ -1159,7 +1249,7 @@ template < int DIM, int SURF > void Update::move()
    return index of sub-cell in ChildCell
 ------------------------------------------------------------------------- */
 
-int Update::split3d(int icell, double *x)
+int Update::split3d(int icell, double *x, int id)
```

Comment:
This is only needed for debug output, did you mean to leave this in?

Reply:
yes, you can whack the extra ID arg to split3d() from every where
that calls it - it was just for temporary debugging.